### ISSUE TITLE
[MDC-Banner] Method isOpen() Is Not a Function

### ISSUE THREAD

**Issue by tonyliunyc** [material-components/material-components-web#7253]
### Steps to reproduce
1. Follow the guidelines to create an `MDCBanner`
2. Call method `isOpen()`:
```js
const banner = new MDCBanner(document.querySelector('.mdc-banner'));
console.log(banner.isOpen())
```
> Uncaught TypeError: banner.isOpen is not a function

### Actual behavior
Throws error

### Expected behavior
Should return boolean (true/false)
Documentation says: 
Method Signature | Description
--- | ---
isOpen() => boolean | Returns whether the banner is open.

### Possible solution
Use the `isOpen` attribute instead -- works perfectly.

**Comment by brandondiamond** [comment]
Hi Tony -- thank you for posting this report. How are you depending on the Material library when reproducing this issue? Can you share a simple codepen/glitch/etc that demonstrates the bug?

A good next step would be to inspect the object produced by `new MDCBanner` to confirm that it is consistent with expectations.

**Comment by EstebanG23** [comment]
Hi Tony thank you for reporting. This is a documentation issue and was fixed by #7279 , where isOpen -like you mentioned under possible solution- is a property and not a method.
